about.html

```diff
@@ -12,24 +12,26 @@
 </head>
 
 <body>
+
+  <img src=img\jumbo-about.png>
     <div class="container about-page">
      
-    About
+    <h1>About</h1>
 
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
     <img src="img/about-plan.png" alt="strategy">
     
```

Comment:
On the about page, there are a couple of key things to keep in mind.  The first is you need to set up your containers around content on the page **and** each section of the content.  The second is keeping in mind that flexbox only nests 2 levels deep, this is very beneficial to the layout of the page.